Lib/queue.py

```diff
@@ -29,9 +30,10 @@ class ShutDown(Exception):
     '''Raised when put/get with shut-down queue.'''
 
 
-_queue_alive = "alive"
-_queue_shutdown = "shutdown"
-_queue_shutdown_immediate = "shutdown-immediate"
+class _QueueState(enum.Enum):
+    ALIVE = "alive"
+    SHUTDOWN = "shutdown"
+    SHUTDOWN_IMMEDIATE = "shutdown-immediate"
```

Comment:
Do we want to introduce a new `enum` import into threading `queue`? It doesn't currently import it.

Reply:
Yes, I would in order to be consistent with asyncio lib

Reply:
There is a comment about to remove enum in 'threading' part.